### ISSUE TITLE
Fedora CoreOS support

### ISSUE THREAD

**Issue by cyrus-mc** [pixie-labs/pixie#252]
**Describe the bug**

I am new to Pixie and decided to give it a try out in a dev cluster. Cluster is provisioned within AWS using EKS version 1.19. Our base OS for worker nodes is Fedora Core OS, specifically version Fedora CoreOS 33.20210201.3.0. 

After running through the installation execution of any script results in 

`
Failed to execute script
Compiler error on line 26, column 22: Table 'xxxxx' not found..
`

List requirements https://docs.px.dev/installing-pixie/requirements/ are met by Fedora CoreOS as the kernel being run is 5.10.2.

**To Reproduce**

1. Install latest Pixie version as documented
2. Load up https://work.withpixie.ai/
3. Execute any script against the cluster

**Expected behavior**

I would expect telemetry data to be returned based on the script executed.

**Screenshots**
If applicable, add screenshots to help explain your problem. Please make sure the screenshot does not contain any sensitive information such as API keys or access tokens.

**Logs**

Logs are attached as instructed.

[pixie_logs_20210506103341.zip](https://github.com/pixie-labs/pixie/files/6436626/pixie_logs_20210506103341.zip)

In addition I see within `vizier-pem` PODS the following error

```
I0506 17:15:06.095028 4004123 linux_headers.cc:89] Obtained Linux version string from `uname`: 5.10.12-200.fc33.x86_64
I0506 17:15:06.095118 4004123 linux_headers.cc:201] Found kernel config at /host/lib/modules/5.10.12-200.fc33.x86_64/config
I0506 17:15:06.101191 4004123 linux_headers.cc:475] Successfully installed packaged copy of headers at /lib/modules/5.10.12-200.fc33.x86_64/build
I0506 17:15:06.101229 4004123 bcc_wrapper.cc:144] Using linux headers found at /lib/modules/5.10.12-200.fc33.x86_64/build for BCC runtime.
I0506 17:15:06.101233 4004123 bcc_wrapper.cc:113] Resolving task_struct offsets.
I0506 17:15:06.482226 4004123 bcc_wrapper.cc:117] Task struct offsets: group_leader=2328 real_start_time=2624
In file included from src/stirling/source_connectors/socket_tracer/bcc_bpf/socket_trace.c:33:
./src/stirling/bpf_tools/bcc_bpf/task_struct_utils.h:63:12: error: no member named 'start_boottime' in 'task_struct'
    return offsetof(struct task_struct, START_BOOTTIME_VARNAME);
           ^                            ~~~~~~~~~~~~~~~~~~~~~~
include/linux/stddef.h:17:32: note: expanded from macro 'offsetof'
#define offsetof(TYPE, MEMBER)  __compiler_offsetof(TYPE, MEMBER)
                                ^                         ~~~~~~
include/linux/compiler_types.h:129:35: note: expanded from macro '__compiler_offsetof'
#define __compiler_offsetof(a, b)       __builtin_offsetof(a, b)
                                        ^                     ~
1 error generated.
E0506 17:15:07.380540 4004123 stirling.cc:320] Source Connector (registry name=socket_tracer) not instantiated, error: Internal : Unable to initialize BCC BPF program: Unable to initialize BPF program
```

Not familiar with the components that make up the Pixie platform yet so not sure what role this component plays. 

**App information (please complete the following information):**
- Pixie version 0.5.8
- k8s cluster version 1.19 (EKS)

**Comment by oazizi000** [comment]
@cyrus-mc Thanks for reporting. This is due to a variable name change in Linux version 5.5. We're working on a fix.

In the meantime, as a workaround, you can try installing linux headers on your Fedora CoreOS host. That will most likely resolve the issue.

**Comment by oazizi000** [comment]
I reproduced and then fixed this issue using an Ubuntu system with a 5.8 kernel and no linux headers.

Fix is in Vizier release 0.7.8.

@cyrus-mc You should try again on the new release.

**Comment by cyrus-mc** [comment]
@oazizi000 I updated and verified that I can now see telemetry data.

Thanks
